pwa/src/components/dashboardCard/DashboardCard.tsx

```diff
@@ -10,33 +10,59 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { dateTime } from "../../services/dateTime";
 import { faArrowRight, faTrash } from "@fortawesome/free-solid-svg-icons";
 import { getStatusTag } from "../../services/getStatusTag";
+import { Button } from "../button/Button";
 
 export interface DashboardCardProps {
   title: {
     label: string;
     href: string;
   };
   type: string;
-  status?: string | boolean;
-  isEnabled?: boolean | undefined;
-  lastRun?: string;
-  lastCall?: string;
-  onDelete?: (e: React.MouseEvent<HTMLButtonElement, MouseEvent> | React.TouchEvent<HTMLButtonElement>) => void;
-}
+  tags: {
+    label: string;
+    tooltip: string;
+  }[];
+  onDelete: (e: React.MouseEvent<HTMLButtonElement, MouseEvent> | React.TouchEvent<HTMLButtonElement>) => void;
 
-export const DashboardCard: React.FC<DashboardCardProps> = ({
-  title,
-  type,
-  status,
-  isEnabled,
-  lastRun,
-  lastCall,
-  onDelete,
-}) => {
-  const { t, i18n } = useTranslation();
+  isEnabled?: boolean;
+}
 
+export const DashboardCard: React.FC<DashboardCardProps> = ({ title, type, tags, isEnabled, onDelete }) => {
   return (
-    <div className={styles.container} onClick={() => navigate(title.href)}>
+    <div className={styles.container}>
+      <div className={styles.headerContainer}>
+        <div>
+          <span>{title.label}</span>
```

Comment:
Maybe this title could have some styling?

E.g. a bold font weight and a bit bigger?

Reply:
It _can_ happen, in the current implementation, that the title is not available, resulting in the following visual:

![image](https://user-images.githubusercontent.com/66552932/220938796-486e4dcb-1f11-4d4e-a1ae-4c7c92b46935.png)

We should write a check, and render some placholder title if none is available.

Reply:
I think it looks good as it is.

This image is with a `font-size` of `18px` and `font-weight` is `normal`
![image](https://user-images.githubusercontent.com/43807324/234530220-6aaeacc0-2a33-46a1-8a14-1e24b8ec8e73.png)

This image is with a `font-size` of `20px` and `font-weight` is `bold`
![image](https://user-images.githubusercontent.com/43807324/234530292-e09a33ba-c90d-4813-974f-29d5e82dab7e.png)